packages/@webex/internal-plugin-device/test/unit/spec/device.js

```diff
@@ -357,6 +357,97 @@ describe('plugin-device', () => {
       });
     });
 
+    describe('deleteDevices()', () => {
+      it('should delete correct number of devices', async () => {
+          const response = {
+            body: {
+                devices: [
+                  {url: 'url3', modificationTime: '2023-10-03T10:00:00Z', deviceType: 'WEB'},
+                  {url: 'url4', modificationTime: '2023-10-04T10:00:00Z', deviceType: 'notweb'},
+                  {url: 'url1', modificationTime: '2023-10-01T10:00:00Z', deviceType: 'WEB'},
+                  {url: 'url2', modificationTime: '2023-10-02T10:00:00Z', deviceType: 'WEB'},
+                ]
+            }
+        };
+
+        const requestStub = sinon.stub(device, 'request');
+        requestStub.withArgs(sinon.match({method: 'GET'})).resolves(response);
+        requestStub.withArgs(sinon.match({method: 'DELETE'})).resolves();
+
+        const clearSpy = sinon.spy(device, 'clear');
+        await device.deleteDevices();
+
+        // Calculate expected deletions
+        const expectedDeletions = ['url1'];
```

Comment:
even though the tests are written in a way that can check that multiple devices are deleted, in all of them the `expectedDeletions` has only 1 entry, so they're not really checking that multiple devices get deleted. It would be nice to increase the number of mocked devices so that we really test that more than 1 is deleted.